src/generator-attributes.cs

```diff
@@ -1012,8 +1019,7 @@ public override string ToString ()
 			GenerateSupported (builder);
 			break;
 		case AvailabilityKind.Deprecated:
-			GenerateAdvice (builder);
```

Comment:
It looks like you can remove the `GenerateAdvice` method now?

Reply:
Done.